cms/envs/common.py

```diff
@@ -47,6 +47,8 @@
 
     # django-debug-toolbar
     DEBUG_TOOLBAR_PATCH_SETTINGS,
+
+    INSTRUCTOR_QUERY_PROBLEM_TYPES,
```

Comment:
it was at the very start of running paver (within seconds). i think the error log mentioned configuration, but I didn't write it down. could check if you're curious